docs/elements/hx-search-control/index.html

```diff
@@ -0,0 +1,132 @@
+---
+title: <hx-search-control>
+minver: 0.18.0
+controlElement: <input type="search">
+also:
+  components/search: Search
+  elements/hx-search: <hx-search>
+  elements/hx-search-assistance: <hx-search-assistance>
+---
+{% extends 'element.njk' %}
+{% import '_utils.njk' as utils %}
+{% block page_header %}
+  <p>
+    The <code>{{page.title}}</code> block element is a container that augments
+    native browser styles, interaction, and validation for an
+    <code>&lt;input type="search"&gt;</code> element.  It is used to build
+    a <a href="components/search">Search</a> component.
+  </p>
+{% endblock %}
+
+{% block content %}
+  <section>
+    <dl class="metadata hxList">
+      <div>
+        <dt>Permitted Parents</dt>
+        <dd>any element that accepts flow content</dd>
+      </div>
+      <div>
+        <dt>Permitted Children</dt>
+        <dd>
+          in this order:
+          <ol class="hxList">
+            <li>
+              {{utils.element('input type="search"')}}
+            </li>
+            <li>
+              {{utils.element('button class="hxClear"')}}
+            </li>
+            <li>
+              {{utils.customElement('hx-search')}}
+            </li>
+            <li>
+              {{utils.element('label')}}
+            </li>
+          </ol>
+        </dd>
+      </div>
+      <div>
+        <dt>Events</dt>
+        <dd><i>none</i></dd>
+      </div>
+    </dl>
+  </section>
+{% endblock%}
+
+{% block attributes %}
+  <dl class="memberList">
+    <dt>
+      hx-changed {Boolean}
+      <hx-pill class="hxSubBody" persist>read-only</hx-pill>
+    </dt>
+    <dd>
+      <p>
+        Present if descendant <code>{{page.controlElement}}</code> element has
+        emitted a <i>change</i> event.
+      </p>
+    </dd>
+    <dt>
+      hx-dirty  {Boolean}
+      <hx-pill class="hxSubBody" persist>read-only</hx-pill>
+    </dt>
+    <dd>
+      <p>
+        Present if descendant <code>{{page.controlElement}}</code> element has
+        emitted a <i>change</i> or <i>blur</i> event.
+      </p>
+    </dd>
+    <dt>
+      hx-touched {Boolean}
+      <hx-pill class="hxSubBody" persist>read-only</hx-pill>
+    </dt>
+    <dd>
+      <p>
+        Present if descendant <code>{{page.controlElement}}</code> element has
+        emitted a <i>blur</i> event.
+      </p>
+    </dd>
+  </dl>
+{% endblock %}
+
+{% block properties %}
+  <dl class="memberList">
+    <dt>
+      controlElement {?HTMLInputElement type="search"}
```

Comment:
This doesn't match formatting of other control elements.

Please remove the `type` portion of the data type (it's already mentioned in the `<dd>` below).

```
controlElement {?HTMLInputElement}
```